locale/en-US/add-on.properties

```diff
@@ -37,6 +37,16 @@ first_time_experience_subheading2=Click the Hello button to browse Web pages wit
 first_time_experience_content=Use it to plan together, work together, laugh together.
 first_time_experience_button_label2=See how it works
 
+## First Time Experience Slides
+fte_slide_1_title=Browse Web pages with a friend
+fte_slide_1_copy=Whether you’re planning a trip or shopping for a gift, Firefox Hello lets you make faster decisions in real time.
```

Comment:
Firefox Hello (and other instances of it) wants to be {{clientShortname2}}, I think.